### DISCUSSION TITLE
How Can i useing match method in micropython_v1.19.1 firmware？

### DISCUSSION THREAD

**Discussion by QiaoTuCodes** [micropython/micropython#10007] [Hardware & Peripherals] [answered]
i try useing match method like that:
```python
def http_error(status):
    match status:
        case 400:
            return "Bad request"
        case 404:
            return "Not found"
        case 418:
            return "I'm a teapot"
        case _:
            return "Something's wrong with the internet"
```
but it's no working. this's a new method of use in Python3.10. we can also add this method? 
look at this way: https://docs.python.org/3/tutorial/controlflow.html?highlight=match%20Statements

**Comment by dlech** [comment, marked as answer]
Please see #7847 and #9850.

**Reply by QiaoTuCodes** [reply]
After reading the results of their discussion, I think I understand!. thank you so much for answer me question.
